Bearing/BearingDoc.cpp

```diff
@@ -72,6 +77,26 @@ BEGIN_MESSAGE_MAP(CBearingDoc, CBEToolboxDoc)
    ON_COMMAND(ID_HELP_FINDER, OnHelpFinder)
 END_MESSAGE_MAP()
 
+
+void CBearingDoc::SetBearing(const Bearing& brg, const BearingLoads& brg_loads)
+{
+	m_bearing = brg;
+	m_bearing_loads = brg_loads;
+	UpdateAllViews(NULL);
+}
+
+void CBearingDoc::GetBearing(Bearing& brg, BearingLoads& brg_loads) const
```

Comment:
Done

Reply:
This morning I was having difficulty getting the program to work properly. It wasn't showing the updated values in the report. It still returns a pair, but also includes the arguments/implementation I had before.

Reply:
Now it works. It returns the data directly from the Document. The reason why the report had bogus data was because of using auto& instead of auto in order to instantiate the bearing/bearing loads objects in the report. See other comment